testing/endpoint.go

```diff
@@ -306,6 +306,10 @@ func (endpoint *Endpoint) ChanOpenTry() error {
 		require.NoError(endpoint.Chain.t, err)
 	}
 
+	// update version to selected app version
+	// NOTE: this update must be performed after the endpoint channelID is set
+	endpoint.ChannelConfig.Version = endpoint.GetChannel().Version
```

Comment:
relayers might need to ensure they are grabbing the channel version set in the OpenTry before calling OpenAck